docs/readthedocs_theme/base.html

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   {% block htmltitle %}
   <title>{{ page_title }}</title>
+  {% if page_description %}<meta name="description" content="{{ page_description }}">{% endif %}
```

Comment:
Nice :) Alright seems to work. Good to know. So the page title itself is shortened now but the page description contains the keywords. Is that OK ? We can also just overwrite the wording in the left-hand navigation block.